surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/info/JUnit5ProviderInfo.java

```diff
@@ -0,0 +1,49 @@
+package org.arquillian.smart.testing.surefire.provider.info;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.arquillian.smart.testing.hub.storage.local.LocalStorage;
+import org.arquillian.smart.testing.surefire.provider.LoaderVersionExtractor;
+
+public class JUnit5ProviderInfo extends JUnitProviderInfo {
+
+    private String junit5SurefirePlatformVersion;
+
+    public JUnit5ProviderInfo() {
+        super(LoaderVersionExtractor.getJUnit5Version());
+        junit5SurefirePlatformVersion = retrieveJunit5SurefirePlatformVersion();
+    }
+
+    public String getProviderClassName() {
+        return "org.junit.platform.surefire.provider.JUnitPlatformProvider";
+    }
+
+    public boolean isApplicable() {
+        return getJunitDepVersion() != null && isAnyJunit5() && junit5SurefirePlatformVersion != null;
```

Comment:
yeah, here we should just check if the `junit5SurefirePlatformVersion` is not null

Reply:
This is what I am doing.

Reply:
I mean that it should check only if the version is present:  
` return junit5SurefirePlatformVersion != null;`
see this comment: https://github.com/arquillian/smart-testing/pull/241#discussion_r147354761